lib/api/decorators/single.rb

```diff
@@ -47,7 +47,13 @@ def initialize(model, context = {})
         super(model)
       end
 
-      property :_type, exec_context: :decorator
+      property :_type,
+               exec_context: :decorator,
+               render_nil: false
+
+      private
+
+      def _type; end
```

Comment:
@ulferts since this class is abstract, I'd suggest `raise NotImplementedError` here.